source/kmwbase.js

```diff
@@ -213,7 +213,8 @@ var __BUILD__ = 300;
       var func=util.events[event][i],result=false;
       try { result=func(params); }
       catch(strExcept) { result=false; }//don't know whether to use true or false here      
-      if(!result) { util.currentEvents.pop(); return false; }
+      if(result === false) {return false;}
+	  else {util.currentEvents.pop();}
```

Comment:
I don't think this is quite correct. I think what we want is:

    if(result === false) { util.currentEvents.pop(); return false; }

Because otherwise the loop is going to possibly pop too many events off the `currentEvent` stack. It may also be important to review the existing event handlers to make sure this doesn't change their behaviour adversely where more than one event handler is registered. I'll check that separately.